plugins/CoreHome/vue/src/ReportExport/ReportExportPopover.vue

```diff
@@ -141,6 +141,8 @@
       </a>
     </div>
 
+    <div v-if="additionalContent" v-html="$sanitize(additionalContent)"></div>
```

Comment:
Should this have some basic css classes like `col l12` or so to keep it at least somewhat contained?

Reply:
guess that makes sense. will also add a new class, so it's possible to select that element directly.